model/src/gx_outp.F90

```diff
@@ -1063,9 +1063,9 @@ SUBROUTINE GXEXPO
 #ifdef W3_ST4
         CALL W3SPR4 (A, CG, WN, EMEAN, FMEAN,  FMEAN1,      &
              WNMEAN, AMAX, UABS, UDIRR,             &
-#ifdef W3_FLX5
+# ifdef W3_FLX5
              TAUA, TAUADIR, RHOAIR,           &
```

Comment:
@MathieuDutSik , are these changes initentional?

Reply:
Update:  I just learned whitespace is allowed between the `#` and `ifdef`.  Out of curiosity then, is there a reason for the whitespace in these particular cases?